src/quote/Quote.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import styles from './Quote.module.css';
+
+const Quote = () => (
+  <div className={styles.quote}>
+    <p>
+      Math magician is not all about numbers, equation,
+      computation or algorithm. it is all about understanding
+      -William paul Thurston
+    </p>
+  </div>
+);
+
+export default Quote;
```

Comment:
- [ ] Kindly move your `Quote Component` to the `component` directory. It is good practice to house all similar files in the same directory. That way your project will be very easy to browse through. In addition, it will make it look more professional and stand out.